src/python/grpcio/grpc/aio/_base_call.py

```diff
@@ -136,7 +136,9 @@ async def wait_for_connection(self) -> None:
 
 
 class UnaryUnaryCall(
-    Generic[RequestType, ResponseType], Call, metaclass=ABCMeta
+    Generic[RequestType, ResponseType],
+    Call,
+    metaclass=ABCMeta,
```

Comment:
Were this changes produced by ruff formatter too? Or the linter? I'm pretty sure black would reformat it into a single line if we get rid of the only non-whitespace change - the trailing comma